eiffel-syntax-and-usage/the-links-object.md

```diff
@@ -25,5 +26,133 @@ __Optional in:__ Any
 __Legal targets:__ [EiffelFlowContextDefinedEvent](../eiffel-vocabulary/EiffelFlowContextDefinedEvent.md)  
 __Description:__ Identifies the flow context of the event: which is the continuous integration and delivery flow in which this occurred – e.g. which product, project, track or version this is applicable to.
 
+### links.activityExecution
+__Type:__ String  
+__Required in:__ [EiffelActivityDequeuedEvent](../eiffel-vocabulary/EiffelActivityDequeuedEvent), 
+[EiffelActivityStartedEvent](../eiffel-vocabulary/EiffelActivityStartedEvent), 
+[EiffelActivityFinishedEvent](../eiffel-vocabulary/EiffelActivityFinishedEvent)  
+__Optional in:__ None  
+__Legal targets:__ [EiffelActivityQueuedEvent](../eiffel-vocabulary/EiffelActivityQueuedEvent.md)  
+__Description:__ Declares the activity execution the event relates to. In other words, [EiffelActivityQueuedEvent](../eiffel-vocabulary/EiffelActivityQueuedEvent.md) acts as a handle for the activity execution.
```

Comment:
The relation between links.activityExecution and links.context needs to be clarified.